src/main/java/school/lemon/changerequest/java/extendedinteger/ExtendedInteger.java

```diff
@@ -1,119 +1,242 @@
 package school.lemon.changerequest.java.extendedinteger;
 
+import java.util.Random;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 /**
  * Integer representation with some additional features
  */
 public class ExtendedInteger {
 
+    int intValue;
+
     public ExtendedInteger(int value) {
-        //TODO: implement me
+        intValue = value;
     }
 
     /**
      * Check whether specified value is even
+     *
      * @param value to check
      * @return true if value is even, false - otherwise
      */
     public static boolean isEven(int value) {
-        //TODO: implement me
-        return false;
+
+        boolean Even = ((value & 1) == 0);
```

Comment:
variable name should start with lower case character. Boolean variables are named as `isSomething`. So in your case it should be named as `isEven`. Or, as it is the returned value, it could have name `result`.